defaults/python/lib/plugin/settings.py

```diff
@@ -152,7 +155,11 @@ def _default_settings(self):
             },
             "enableMoondeckShortcuts": True,
             "enableMoondeckButtonPrompt": False,
-            "hostSettings": {},
```

Comment:
This should be kept as is, because it's filled in on frontend side with default values.